espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/drivers/ComposeDriver.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.drivers
+
+import androidx.compose.ui.semantics.SemanticsProperties
+import androidx.compose.ui.test.*
+import io.appium.espressoserver.lib.helpers.ComposeNodeFinder
+import io.appium.espressoserver.lib.model.*
+
+class ComposeDriver : AppDriver {
+    override val name = DriverContext.StrategyType.COMPOSE
+
+    override fun findElement(params: Locator): BaseElement {
+
+        var parentNodeInteraction: SemanticsNodeInteraction? = null
+
+        params.elementId?.let {
+            parentNodeInteraction = ComposeElement.getNodeInteractionById(it)
+        }
+
+        val nodeInteractions = ComposeNodeFinder.findAllComposeNodeInteractions(parentNodeInteraction, params)
+
+        return ComposeElement(nodeInteractions[0])
+    }
+
+
+
+    override fun click(params: AppiumParams): Void? {
+        val nodeInteraction = ComposeElement.getNodeInteractionById(params.elementId)!!
```

Comment:
the result of `getNodeInteractionById` should not be nullable. See the comment below

Reply:
fixed